app/src/common/settings-manager.js

```diff
@@ -14,7 +14,13 @@ const DEFAULTS = {
   highlightClicks: false,
   fps: 30,
   recordAudio: false,
-  audioInputDeviceId: 'none'
+  audioInputDeviceId: 'none',
+  dimensions: {
+    height: 512,
+    width: 512,
+    ratio: '1:1',
```

Comment:
I think we should persist it as plain numbers. I know it's already represented as a string in the code, but the code we can change later. It's much harder to change the settings.

So it should be saved as `ratio: [1, 1]`

Reply:
I can do that. Shouldn't be too big of a change. Do you want it as part of this PR?

Reply:
Sure

Reply:
600a667